src/app/internarbeidsflatedecorator/Decorator.tsx

```diff
@@ -1,25 +1,78 @@
 import * as React from 'react';
-import { useEffect } from 'react';
-import renderDecoratorHead from './decoratorconfig';
+import { useCallback } from 'react';
+import NAVSPA from '@navikt/navspa';
+import { History } from 'history';
 import { AppState } from '../../redux/reducers';
 import { connect } from 'react-redux';
+import { DecoratorProps } from './decoratorprops';
+import { getSaksbehandlerEnhet } from '../../utils/loggInfo/saksbehandlersEnhetInfo';
+import { apiBaseUri } from '../../api/config';
+import { fjernBrukerFraPath, setNyBrukerIPath } from '../routes/routing';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { isDevelopment } from '../../utils/environment';
+import { finnMiljoStreng } from '../../utils/url-utils';
 
-interface StateProps {
-    fødselsnummer: string;
+interface StateProps extends RouteComponentProps<{}> {
+    fnr: string | null;
+    enhet: string | undefined;
 }
 
-function Decorator(props: StateProps) {
-    useEffect(() => {
-        renderDecoratorHead(props.fødselsnummer);
-    }, [props.fødselsnummer]);
+const InternflateDecorator = NAVSPA.importer<DecoratorProps>('internarbeidsflatefs');
+const identSource = () => fetch(`${apiBaseUri}/hode/me`).then(resp => resp.json());
+const enheterSource = () => fetch(`${apiBaseUri}/hode/enheter`).then(resp => resp.json());
 
-    return <nav id="header" />;
+function lagConfig(fnr: string | undefined | null, enhet: string | undefined | null, history: History): DecoratorProps {
+    return {
+        appname: 'Modia personoversikt',
+        fnr,
+        enhet,
+        toggles: {
+            visEnhet: false,
+            visEnhetVelger: true,
+            visSokefelt: true,
+            visVeilder: true
+        },
+        identSource,
```

Comment:
kunne `identsource` og `enhetssource` ligget i internflatedecoratoren?

Reply:
Tror vi vil skape litt problemer om vi gjør det. 
Hvilken url skal vi bruke liksom? :P 

For `identsource` så kunne vi sikkert lagt til noe i modia-contextholder, men `enhetersource` er det verre med siden en saksbehandler kan ha tilgang til forskjellige enheter basert på hvilket system det er snakk om.